### DISCUSSION TITLE
Transform YouTube Live links into streamable IPTV

### DISCUSSION THREAD

**Discussion by damare01** [iptv-org/community#583] [Q&A]
Is it possible to turn YouTube Live links into streamable IPTV? For instance this Philippine based entertainment program has a YouTube channel "https://m.youtube.com/abscbnentertainment/live" so that people can also watch them on their devices.

**Comment by METALGROBICK** [comment]
maybe this will help you

https://github.com/abskmj/iptv-youtube-live
